README.md

```diff
@@ -1 +1,137 @@
-# oop-school-library
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b>School Library</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [School-Library] <a name="School Library<"></a>
+
+**School Library** is a project in wich I will build a school library app
+
```

Comment:
- [ **OPTIONAL** ] I think it would be good if you improve your project description to make it more professional. You can go to your sneak peak the project from your dashboard to get the meaningful description of the project. I know you can do it :muscle: :muscle: